pipeline/beam_tables.py

```diff
@@ -194,24 +194,13 @@ def _read_scan_text(
     A PCollection[Tuple[filename, line]] of all the lines in the files keyed
     by filename
   """
-  # List[PCollection[Tuple[filename,line]]]
-  line_pcollections_per_file: List[beam.PCollection[Tuple[str, str]]] = []
+  # PCollection[filename]
```

Comment:
I don't think you need `_read_scan_text`. The indirection hurts readability. I'd just inline the code in the caller.

Reply:
I'd like to keep this seperate for the purposes of keeping the unit test below. If it's inlined in the caller then we can't test this functionality independently.

Reply:
Generally anything that gets inlined into `run_beam_pipeline` becomes untestable outside of the e2e tests since `run_beam_pipeline` writes to bigquery.